Sources/Bot/Services/MergeService.swift

```diff
@@ -279,6 +279,27 @@ extension MergeService {
     }
 }
 
+extension MergeService.Event.Outcome {
```

Comment:
Do we really need a type for this? Why not using free/static functions? I don't find it a type particularly discoverable 😕

Reply:
When I say a type I mean using failable initialisers.

Reply:
tbh I initially created a `static` free function at first, but the signature felt a bit strange to read, especially since I named that `static func outcome(for: …) -> MergeService.Event.Outcome?`… so since the func was named `outcome` and returned the (already existing †) `Outcome` type, I figured it would make more sense to make it an initialiser on the existing type

_(†) The `Outcome` type is not new and used in the `Feedbacks` of the `MergeService` machine already, all I did is move the logic from the `Feedback` to a `init?` to be able to _also_ call it from the `DispatchService` to decide wether to create a new queue or not)_

I'm ok with moving those to free functions if you prefer it to using an `init?` – just know that even if I do that the `MergeService.Event.Outcome` type will still exist as it's used in the state machine already.

Reply:
That's ok let's keep it, is just a detail 👍